### PR TITLE
Update extension validation policies

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: src/AppBundle/Resources/views/Extensions/validation-policy.html.twig [hunk 1]
```diff
@@ -48,10 +48,12 @@
 			</li>
 			<li id="general-lang-styles">
 				<h3>Language and Styles</h3>
-				Extensions including language files and/or style files should always provide files for the English language and Prosilver style, respectively. <a class="permalink" href="#general-lang-styles">#</a>
+				Language files and/or style files should always provide files for the English language and Prosilver style, respectively. <a class="permalink" href="#general-lang-styles">#</a>
```
Comment (paul999) [inline src/AppBundle/Resources/views/Extensions/validation-policy.html.twig]: Lower case Prosilver.

**Comment by VSEphpbb** [comment]
@paul999 By Grabthar's Hammer, your wish is my command.

**Comment by DavidIQ** [comment]
@michaelcullum needs merging too

**Comment by michaelcullum** [comment]
@DavidIQ Sure. Was waiting for yours or Paul's 👍

**Pull request merged**